substrate/frame/support/test/tests/pallet.rs

```diff
@@ -14,7 +14,7 @@
 // WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 // See the License for the specific language governing permissions and
 // limitations under the License.
-#![allow(useless_deprecated, deprecated, clippy::deprecated_semver)]
+#![allow(useless_deprecated, clippy::deprecated_semver)]
```

Comment:
why is deprecated_semver needed by the way?
Also people will have to bound useless_deprecated, or we can remove it?

Reply:
there were tests during the initial review with non-semver strings in the deprecation attribute. removed all of allows above